docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/KeyToValueToKey.cs

```diff
@@ -13,83 +13,65 @@ public static void Example()
+                keyOrdinality: ValueToKeyMappingEstimator.KeyOrdinality.ByValue));
 
             // The transformed data.
-            var transformedData_default = default_pipeline.Fit(trainData).Transform(trainData);
-            var transformedData_customized = customized_pipeline.Fit(trainData).Transform(trainData);
-
-            // Small helper to print the text inside the columns, in the console. 
-            Action<string, IEnumerable<VBuffer<uint>>> printHelper = (columnName, column) =>
-            {
-                Console.WriteLine($"{columnName} column obtained post-transformation.");
-                foreach (var row in column)
-                {
-                    foreach (var value in row.GetValues())
-                        Console.Write($"{value} ");
-                    Console.WriteLine("");
-                }
-
-                Console.WriteLine("===================================================");
-            };
-
-            // Preview of the DefaultKeys column obtained after processing the input.
-            var defaultColumn = transformedData_default.GetColumn<VBuffer<uint>>(transformedData_default.Schema[defaultColumnName]);
-            printHelper(defaultColumnName, defaultColumn);
-
-            // DefaultKeys column obtained post-transformation.
-            //
-            // 1 2 3 4 5 6
-            // 6 2 7 5 8 3
-            // 9 10 11 12 13 3
-            // 9 10 11 12 13 6
-
-            // Previewing the CustomizedKeys column obtained after processing the input.
-            var customizedColumn = transformedData_customized.GetColumn<VBuffer<uint>>(transformedData_customized.Schema[customizedColumnName]);
-            printHelper(customizedColumnName, customizedColumn);
-
-            // CustomizedKeys column obtained post-transformation.
-            //
-            // 1 2 4 5 7 8
-            // 8 2 9 7 6 4
-            // 3 10 0 0 0 4
-            // 3 10 0 0 0 8
-
+            var transformedDataDefault = defaultPipeline.Fit(trainData).Transform(trainData);
+            var transformedDataCustomized = customizedPipeline.Fit(trainData).Transform(trainData);
+
+            // Getting the resulting data as an IEnumerable.
+            // This will contain the newly created columns.
+            IEnumerable<TransformedData> defaultData = mlContext.Data.CreateEnumerable<TransformedData>(transformedDataDefault, reuseRowObject: false);
+            IEnumerable<TransformedData> customizedData = mlContext.Data.CreateEnumerable<TransformedData>(transformedDataCustomized, reuseRowObject: false);
+            Console.WriteLine($"Keys");
+            foreach (var dataRow in defaultData)
+                Console.WriteLine($"{string.Join(',', dataRow.Keys)}");
+            // Expected output:
+            //  Keys
+            //  1,2,3,4,5,6
+            //  6,2,7,5,8,3
+            //  9,10,11,12,13
+            //  9,10,11,12,13,6
```

Comment:
please verify, if input data has changed.

Reply:
I got them from my console output.

---
In reply to: [277895084](https://github.com/dotnet/machinelearning/pull/3539#discussion_r277895084) [](ancestors = 277895084)